### PR TITLE
Update Azure-pipelines-task-lib version in Azure App Service Manage task

### PR DESCRIPTION
**Pull request by 20shivangi**

### CODE REVIEW

File: Tasks/AzureAppServiceManageV0/Tests/package-lock.json [hunk 1]
```diff
@@ -16,9 +16,9 @@
       "integrity": "sha1-TQJjewZ/6Vi9v906QOxW/vc3Mkc=",
```
Comment (vincentdass) [inline Tasks/AzureAppServiceManageV0/Tests/package-lock.json]: https://github.com/Microsoft/azure-pipelines-tasks/blob/222d274f34766a2a347799172f5080ba74ec8ab6/Tasks/AzureAppServiceManageV0/package-lock.json

File: Tasks/AzureAppServiceManageV0/package-lock.json [hunk 2]
```diff
@@ -34,11 +34,11 @@
       "integrity": "sha512-PR8oap9z2j+o455W3PwAfB4SX1p4GdJc9OHQaQV0V+iQS1IBY6dVgcNSQMkHAXb0V1bbuLOFBLanXPe5eSgGTQ==",
```
Comment (vincentdass) [inline Tasks/AzureAppServiceManageV0/package-lock.json]: https://github.com/Microsoft/azure-pipelines-tasks/blob/222d274f34766a2a347799172f5080ba74ec8ab6/Tasks/AzureAppServiceManageV0/package-lock.json

**Comment by vincentdass** [comment]
git clean -fdx
npm install
git pull
node make.js build test --task AzureAppServiceManageV0
--tfx command to upload ---

File: Tasks/AzureAppServiceManageV0/azureappservicemanage.ts [hunk 3]
```diff
@@ -16,6 +16,7 @@ import { enableContinuousMonitoring } from './operations/ContinuousMonitoringUti
 async function run() {
     try {
         tl.setResourcePath(path.join( __dirname, 'task.json'));
+		tl.setResourcePath(path.join( __dirname, 'module.json'));
```
Comment (vincentdass) [inline Tasks/AzureAppServiceManageV0/azureappservicemanage.ts]: `node_modules\azure-arm-rest-v2\module.json`

File: Tasks/AzureAppServiceManageV0/azureappservicemanage.ts [hunk 4]
```diff
@@ -16,6 +16,7 @@ import { enableContinuousMonitoring } from './operations/ContinuousMonitoringUti
 async function run() {
     try {
         tl.setResourcePath(path.join( __dirname, 'task.json'));
+		tl.setResourcePath(path.join( __dirname, 'node_modules/azure-arm-rest-v2/module.json'));
```
Comment (vincentdass) [inline Tasks/AzureAppServiceManageV0/azureappservicemanage.ts]: indentation

File: Tasks/Common/azure-arm-rest-v2/package.json [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    "url": "git+ssh://git@github.com/Microsoft/azure-pipelines-tasks.git"
+  },
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Microsoft/azure-pipelines-tasks/issues"
+  },
+  "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
+  "dependencies": {
+    "jsonwebtoken": "7.3.0",
```
Comment (vincentdass) [inline Tasks/Common/azure-arm-rest-v2/package.json]: "@types/mocha": "2.2.48",
    "@types/node": "6.0.68",
    "@types/q": "1.0.7",

File: package-lock.json [hunk 6]
```diff
@@ -892,4 +892,4 @@
       "dev": true
     }
   }
-}
```
Comment (SumiranAgg) [inline package-lock.json]: remove this file from pr

File: Tasks/AzureAppServiceManageV0/azureappservicemanage.ts (hunk 3 above)
Comment (20shivangi) [reply Tasks/AzureAppServiceManageV0/azureappservicemanage.ts]: it is done

File: Tasks/AzureAppServiceManageV0/azureappservicemanage.ts (hunk 4 above)
Comment (20shivangi) [reply Tasks/AzureAppServiceManageV0/azureappservicemanage.ts]: it is done

File: Tasks/Common/azure-arm-rest-v2/package.json (hunk 5 above)
Comment (20shivangi) [reply Tasks/Common/azure-arm-rest-v2/package.json]: it is done.

File: package-lock.json (hunk 6 above)
Comment (20shivangi) [reply package-lock.json]: it is done.

**Pull request merged**